agent/src/main/resources/build-scan-init.gradle

```diff
@@ -201,6 +201,20 @@ static boolean isNotAtLeast(String versionUnderTest, String referenceVersion) {
     GradleVersion.version(versionUnderTest) < GradleVersion.version(referenceVersion)
 }
 
+void applyPluginExternally(PluginManager pluginManager, String pluginClassName) {
```

Comment:
Would you mind moving this non-static method to above `extensionsWithPublicType` ?

Reply:
Done in b244b2f8238501c353fc8bd56dee722770e30259